download.py

```diff
@@ -5,9 +5,47 @@
 import logging
 from transformers import CLIPProcessor, CLIPModel
 from sentence_transformers import SentenceTransformer
+import open_clip
+import json
 
 logging.basicConfig(level=logging.INFO)
 
+open_clip_model_name = os.getenv('OPEN_CLIP_MODEL_NAME')
+open_clip_pretrained = os.getenv('OPEN_CLIP_PRETRAINED')
+
+if open_clip_model_name != "" and open_clip_pretrained != "":
+  def check_model_and_pretrained(model_name: str, pretrained: str):
```

Comment:
You can make this function a bit shorter and faster:
```python
def check_model_and_pretrained(model_name: str, pretrained: str):
   if (model_name, pretrained) in open_clip.list_pretrained():
      return
   logging.error("Fatal: Available pairs are:")
   for pair in open_clip.list_pretrained():
      logging.error(f"Fatal: model: {pair[0]} pretrained: {pair[1]}")
   logging.error(f"Fatal: Match not found for OPEN_CLIP model {model_name} with pretrained {pretrained} pair")
   sys.exit(1)
```

Reply:
nice! thanks